### PR TITLE
Fix Android 6.1.0 Crashing

### PR DESCRIPTION
**Pull request by redreceipt**
Fixes [SYS-4545 Android Crashing from Firebase Call](https://newspring.atlassian.net/browse/SYS-4545)

## Creator

- [ ] Build relevant tests if any apply
- [ ] Ensure there are no new warnings
- [ ] Upload an animated GIF showcasing the solution (if it applies)
- [ ] Set a relevant reviewer

## Reviewer

- [ ] Run `test` and make sure all tests pass
- [ ] Review function and form on iOS
- [ ] Review function and form on Android
- [ ] Review function and form on Web
- [ ] Review new test logic

### CODE REVIEW

**Comment by IsaacHardy** [comment]
@redreceipt Can you elaborate a bit on these changes? What are they doing? Why did this change need to be made? Etc..

**Comment by redreceipt** [comment]
@IsaacHardy this fixes a Firebase lib conflict described here
https://stackoverflow.com/questions/46340333/unable-to-start-receiver-com-google-firebase-iid-firebaseinstanceidinternalrecei

From my vague understanding, the app is crashing because the older Firebase lib starts a service in the background which conflicts with Android 8.0.  It was set to Firebase-9.8.0 from ExpoKit. Updating to Firebase-10.2.1 has fixed issues for a lot of people but if that doesn't work, we could try the more complicated Android Intent fix in the described solution here:
https://stackoverflow.com/questions/51532059/android-gcm-wakefulbroadcastreceiver-startwakefulservice-crash-due-to-illegals

Lastly, if all that doesn't work we could roll back the build to SDK 25 (Android v7) but I don't remember why we had to upgrade it so that could cause adverse other adverse effects. Currently, we're on 26.

**Comment by IsaacHardy** [comment]
@redreceipt Why are we currently using firebase?

**Comment by redreceipt** [comment]
We’re not, react-native-maps is

**Comment by IsaacHardy** [comment]
Are we using react-native-maps?

**Comment by redreceipt** [comment]
Might not be exactly `react-native-maps` but it's some React Native library that uses an old Firebase lib. ExpoKit imports a bunch of libraries that we probably don't use in our code but they do.

File: android/app/build.gradle [hunk 1]
```diff
@@ -21,7 +21,7 @@ android {
 
   defaultConfig {
     applicationId 'cc.newspring.newspringapp'
-    targetSdkVersion 26
+    targetSdkVersion 25
```
Comment (IsaacHardy) [inline android/app/build.gradle]: https://developer.android.com/distribute/best-practices/develop/target-sdk

Looks like this won't work...

**Pull request merged**